IaC/lambda_api_module/main.tf

```diff
@@ -0,0 +1,154 @@
+locals{
+  API_zipName = "../${api_name}/bin/Release/net6.0/${api_name}.zip"
+  APIFunctionName = "${api_name}Lambda"
```

Comment:
Since this is only used once, you can just put the value there instead of using a local.